recipes/pywps/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "pywps" %}
+{% set version = "4.0.0" %}
+
+package:
+    name: {{ name|lower }}
+    version: {{ version }}
+
+source:
+    git_url: https://github.com/geopython/pywps
+    git_tag: {{ version }}
+
+build:
+    number: 0 
+    skip: True  # [win]
+    script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+    build:
+        - python
+        - setuptools
+        - owslib
```

Comment:
When using setuptools and `--single-version-externally-managed` you probably don't need the `run` dependencies at build time. Try removing,

```
        - owslib
        - jsonschema
        - werkzeug
        - lxml
        - sqlalchemy
        - flufl.enum  # [py27]
        - python-dateutil
```

PS: if the package imports itself in the `setup.py` that may not be true and you will need to leave those there.